config/crd/bases/infrastructure.cluster.x-k8s.io_awsclustercontrolleridentities.yaml

```diff
@@ -104,6 +104,7 @@ spec:
                           "value". The requirements are ANDed.
                         type: object
                     type: object
+                    x-kubernetes-map-type: atomic
```

Comment:
Is this SSA related parameter?

Reply:
These are auto generated files, this change is due to dependency bump.